### DISCUSSION TITLE
unable to implement pagination with multiple tables in same page

### DISCUSSION THREAD

**Discussion by aditya4kasi** [primefaces/community#1242] [PrimeNG]
unable to implement pagination with multiple tables in same page, 
able to load 3 different tables data in same page, but the pagination was not able to implement differently for each table. also the sorting and filter functionalities not working.

in package.json using below

"dependencies": {
    "@angular/animations": "^15.2.0",
    "@angular/cdk": "^15.2.8",
    "@angular/common": "^15.2.0",
    "@angular/compiler": "^15.2.0",
    "@angular/core": "^15.2.0",
    "@angular/forms": "^15.2.0",
    "@angular/platform-browser": "^15.2.0",
    "@angular/platform-browser-dynamic": "^15.2.0",
    "@angular/router": "^15.2.0",
    "primeng": "^15.4.1",
    "font-awesome": "^4.7.0",
    "primeicons": "^6.0.1",
    "primeflex": "^3.2.1",
    "rxjs": "~7.8.0",
    "tslib": "^2.3.0",
    "zone.js": "~0.12.0"
  }

my html component code looks like as below: 

 <p-table #dt [columns]="cols" [value]="apps" [rows]="10"
                [rowsPerPageOptions]="[10,15,25]" [first]="first1"
                [paginator]="true" [totalRecords]="apps.length">

 <p-table #dtg [columns]="gcols" [value]="grans" [rows]="10"
                [rowsPerPageOptions]="[10,15,25]" [first]="first2"
                [paginator]="true" [totalRecords]="grans.length">

 <p-table #dtm [columns]="mcols" [value]="files" [rows]="10"
                [rowsPerPageOptions]="[10,15,25]" [first]="first3"
                [paginator]="true" [totalRecords]="files.length">

Please help me here.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Comment by aditya4kasi** [comment]
I fixed it, Thank you.
